### PR TITLE
New version: KristenMcWilliam.UnitBargainHunter version 1.1.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58043)

### CODE REVIEW

File: manifests/k/KristenMcWilliam/UnitBargainHunter/1.1.0/KristenMcWilliam.UnitBargainHunter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: KristenMcWilliam.UnitBargainHunter
+PackageVersion: 1.1.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
```
Comment (OfficialEsco) [inline manifests/k/KristenMcWilliam/UnitBargainHunter/1.1.0/KristenMcWilliam.UnitBargainHunter.installer.yaml:7]:
```suggestion
InstallerType: msix
```

File: manifests/k/KristenMcWilliam/UnitBargainHunter/1.1.0/KristenMcWilliam.UnitBargainHunter.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Scope: user
+ReleaseDate: 2022-04-17
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Merrit/unit_bargain_hunter/releases/download/v1.1.0/Unit.Bargain.Hunter-Windows-Installer.msix
+  InstallerSha256: 654BCE0F1DADCCC68030924B01AE8120F9C7711219BF6935AC7A892852E8A024
```
Comment (OfficialEsco) [inline manifests/k/KristenMcWilliam/UnitBargainHunter/1.1.0/KristenMcWilliam.UnitBargainHunter.installer.yaml:13]:
```suggestion
  InstallerSha256: 654BCE0F1DADCCC68030924B01AE8120F9C7711219BF6935AC7A892852E8A024
  SignatureSha256: 7D3BD7AD052B44B48880EFA1C8A35AB24E7AC302DBA13ECE8FB40DCC5FBFEDBC
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
However there is also this:
```
Add-AppxPackage: Deployment failed with HRESULT: 0x80073CF0, Package could not be opened.
error 0x800B0109: The root certificate of the signature in the app package or bundle must be trusted.
NOTE: For additional information, look for [ActivityId] 97811150-52fd-0000-636f-8397fd52d801 in the Event Log or use the command line Get-AppPackageLog -ActivityID 97811150-52fd-0000-636f-8397fd52d801
```
>This app package’s publisher certificate could not be verified. Contact your system administrator or the app developer to obtain a new app package with verified certificates. The root certificate and all immediate certificates of the signature in the app package must be verified (0x800B010A)

**Comment by Trenly** [comment]
```
2022-04-18 10:18:39.014 [CORE] Deployment operation #1: Windows cannot remove codes.merritt.bargain_1.1.0.0_x64__fxkeb4dgdm144 because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.
2022-04-18 10:18:39.014 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\AppInstallerCLI.exe!00007FF666199124: (caller: 00007FF66619A6E5) Exception(2) tid(1038) 80073CF1 Package was not found.

    Msg:[Operation failed: Windows cannot remove codes.merritt.bargain_1.1.0.0_x64__fxkeb4dgdm144 because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.] 

2022-04-18 10:18:39.014 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(124)\AppInstallerCLI.exe!00007FF6662E022E: (caller: 00007FF66619A783) LogHr(1) tid(1038) 80073CF1 Package was not found.

    Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\AppInstallerCLI.exe!00007FF666199124: (caller: 00007FF66619A6E5) Exception(2) tid(1038) 80073CF1 Package was not found.

    Msg:[Operation failed: Windows cannot remove codes.merritt.bargain_1.1.0.0_x64__fxkeb4dgdm144 because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.] 
] 

2022-04-18 10:18:39.028 [CLI ] MSIX installer failed: 2148204809
```

**Pull request closed without merging**